### ISSUE TITLE
does not work with strings, strange behaviour when ${} variables present in string literals

### ISSUE THREAD

**Issue by numeralz** [ipatalas/vscode-postfix-ts#48]
```json
  "postfix.customTemplates": [
    {
      "body": "try{\n  const {\n    data,\n  } = await axios.request( {\n    method : \"${1:get}\",\n    url    : {{expr}},\n  } );\n  // handle result\n  $2\n}\ncatch( error ){\n  console.error( error );\n}",
      "description": "AxiosReq",
      "name": "axiosget",
      "when": []
    }
  ]
```

```ts
const userId=1;
const postId=1;
`/api/user/${userId}/posts/${postId}`.axiosget
```
expansion produces:
```ts
`/api/user/${userId}/posts/${try{
  const {
    data,
  } = await axios.request( {
    method : "get",
    url    : postId}`,
  } );
  // handle result
  
}
catch( error ){
  console.error( error );
}
```

expected:
```ts

try{
  const {
    data,
  } = await axios.request( {
    method : "get",
    url    : `/api/user/${userId}/posts/${postId}`,
  } );
  // handle result
  
}
catch( error ){
  console.error( error );
}
```

actually, further investigation shows that it does not really work for strings.

**Comment by ipatalas** [comment]
Hi, thanks for reporting. It really doesn't work for strings. That's amazing I haven't noticed that before :D
I'll have a look into that.

**Comment by Capyhw** [comment]
does this resolved？

**Comment by ipatalas** [comment]
Hi, sorry it took so long but got it figured out:
![image](https://user-images.githubusercontent.com/2623329/169691267-7c9b1cd5-c631-4fb4-86db-1e85d0fc031a.png)

There is one more thing I'd like to do before next release but I hope it will happen in the upcoming days.

**Comment by ipatalas** [comment]
Fixed in 1.10.0
